frida_tools/application.py

```diff
@@ -70,6 +70,22 @@ def await_enter(reactor):
         print("")
 
 
+def deserialize_relay(value):
+    address, username, password, kind = value.split(",")
+    return frida.Relay(address, username, password, kind)
+
+
+def create_target_parser(type):
+    def parse_target(value):
+        if type == "file":
+            value = [value]
+        elif type == "gated":
+            value = re.compile(value)
+        elif type == "pid":
```

Comment:
We currently use single-quoted strings for enum/symbol values, and double-quoted for everything else. So would be good to use that here too.

Reply:
Fixed but a weird convention isn't it? Maybe it would be better to create the appropriate enums in the Python bindings?

Reply:
Yeah I agree. Would be good to switch all quotes to one style. (Whichever you prefer, I'm fine with either.)